DoSomethingArchive/legacy-website
# Move international sites (Indonesia) to Docker and add Pantheon integration

#### What's this PR do?
This PR does a few things:

- Switches to Docksal (Docker-based) for local environments
- Has new scripting to build the codebase locally and clone the DB
- Includes a script to push the current codebase to Pantheon's dev environment

#### How should this be reviewed?
1. [Install Docksal](http://docksal.readthedocs.io/en/master/getting-started/env-setup-native/#using-docksal-with-docker-for-macwindows-new-users)
2. Check out this feature branch
3. Run `fin init` and make sure nothing breaks and that at the end you have a working local site.
